src/nupic/bindings/PySparseTensor.cpp

```diff
@@ -1,115 +0,0 @@
-/* ---------------------------------------------------------------------
```

Comment:
deleted by mistake? Would probably be used

Reply:
> deleted by mistake?

Oh, I grouped them by algorithm, engine, and math folders so they looked better in MSVC.  So they were not deleted, just moved.